clang/include/clang/Basic/AttrDocs.td

```diff
@@ -3612,6 +3612,22 @@ an error:
 }];
 }
 
+def MSConstexprDocs : Documentation {
+  let Category = DocCatStmt;
+  let Content = [{
+The ``[[msvc::constexpr]]`` attribute can be applied only to a function
+definition or a ``return`` statement. It does not impact function declarations.
+A ``[[msvc::constexpr]]`` function cannot be ``constexpr`` or ``consteval``.
+A ``[[msvc::constexpr]]`` function is treated in the same way as a ``constexpr``
```

Comment:
```suggestion
A ``[[msvc::constexpr]]`` function is treated as if it were a ``constexpr``
```

Reply:
Thanks, applied manually. Note, now there are two `if` words nearby. But perhaps it's still better.